internal/config/config.go

```diff
@@ -97,10 +97,12 @@ var (
 	CorsEnabled = rootKey("cors.enabled")
 	// CorsMaxAge is the maximum age a browser should rely on CORS checks
 	CorsMaxAge = rootKey("cors.maxAge")
-	// DataexchangeType is the name of the data exchange plugin being used by this firefly name
+	// DataexchangeType is the name of the data exchange plugin being used by this firefly node
 	DataexchangeType = rootKey("dataexchange.type")
 	// DatabaseType the type of the database interface plugin to use
 	DatabaseType = rootKey("database.type")
+	// TokensType is the name of the tokens plugin being used by this firefly node
+	TokensType = rootKey("tokens.type")
```

Comment:
This implies a configuration of the form:
```
tokens:
  type: https
  https:
    url: http://localhost:5108
```

I don't think "type" is actually the right word here, and I believe we want to support multiple types of tokens in a single config. I'm working follow-up changes in this area, possibly toward something like this (but open to ideas):
```
tokens:
  - connector: https
    name: erc1155
    url: http://localhost:5108
  - connector: https
    name: erc20
    url: http://localhost:5109
```

Reply:
Added a commit to make this change.